src/main/java/org/broadinstitute/hellbender/tools/spark/sv/evidence/FindSmallIndelRegions.java

```diff
@@ -114,49 +253,140 @@ public Finder( final ReadMetadata readMetadata, final SVReadFilter filter ) {
+                    statusList.add(String.format("%s\t%s\t%s\t%d\t%d\t0",
+                                            libName, overlapperItr.hasNext()?"FN":"TN", curContig, start, curEnd));
+                else {
+                    final LibraryStatistics stats = readMetadata.getLibraryStatistics(libName);
+                    final IntHistogram.CDF cdf = stats.getCDF();
+                    final IntHistogram.CDF.KSData ksData = cdf.evalKS(oldHisto);
+                    final long maxObservations = (long)(MAX_OBSERVATIONS_SCALE*stats.getReadStartFrequency());
+                    final boolean isCalledInterval =
+                            ksData.getSignificance() <= KS_SIGNIFICANCE &&
+                            oldHisto.getTotalObservations() <= maxObservations;
+                    final boolean isTrue = overlapperItr.hasNext();
+                    final String truthStatus = (isCalledInterval==isTrue ? "T" : "F") +
+                            (isCalledInterval ? "P" : "N");
+                    final int relativeLocation = stats.getMedian() - ksData.getLocationOfMaxDiff();
+                    final int firstOverlappingEventSize;
+                    if ( !overlapperItr.hasNext() ) firstOverlappingEventSize = 0;
+                    else {
+                        final SVTypeLen overlappingTypeLen = overlapperItr.next().getValue();
+                        switch ( overlappingTypeLen.getType() ) {
+                            case DEL: firstOverlappingEventSize = -overlappingTypeLen.getLen(); break;
+                            case INS: firstOverlappingEventSize = overlappingTypeLen.getLen(); break;
+                            default: firstOverlappingEventSize = 0; break;
+                        }
+                    }
+                    final String status = String.format("%s\t%s\t%s\t%d\t%d\t%d\t%d\t%d\t%.2f\t%.2f\t%.1e",
+                                                        libName, truthStatus, curContig, start, curEnd, counts,
+                                                        relativeLocation, firstOverlappingEventSize,
+                                                        ksData.getMaxDiff(), ksData.getMaxArea(),
+                                                        ksData.getSignificance());
+                    statusList.add(status);
+                    if ( cdfDir != null && (isCalledInterval || isTrue) ) {
+                        final String fileName =
+                                cdfDir + "/" + libName + "." + curContig + "." + start + "." + truthStatus + ".cdf";
+                        dumpCDF(fileName, oldHisto.getCDF(), status, trueIntervals.overlappers(curInterval));
+                    }
                 }
                 oldHisto.clear();
             }
-            fillIdx = oldIdx;
+        }
+
+        private void advanceBlock( final boolean sameContig, final GATKRead read ) {
+            final int oldIdx = fillIdx;
+            fillIdx ^= 1; // bit magic to switch halves
```

Comment:
This bit magic that affects `fillIdx` is executed in the master version only conditionally: i.e. when `sameContig` is true. Not sure if this change is intended.